src/main/java/edu/cornell/kfs/rass/batch/service/impl/RassReportServiceImpl.java

```diff
@@ -284,6 +284,19 @@ private void writeHeaderForResultsDetailsByResultCode(Class businessObjectClass,
 
         reportWriterService.writeFormattedMessageLine(reportMessage);
     }
+    
+    private String buildErrorLines(String prefix, String errorMessage) {
+        StringBuffer errorLines = new StringBuffer();
```

Comment:
Unless you need multi-thread appending or need to work with APIs that only accept StringBuffer, I would suggest using a StringBuilder instead.

Reply:
updated